scripts/dist2xml.py

```diff
@@ -0,0 +1,9 @@
+import json
```

Comment:
It seems like we have a BUNCH of scripts these days, and I know know what most of them do, or even if they're currently used. I would suggest:

1. Create a `contents.rst` file which gives a one-sentence description of each scritp.
2. Eventually, port scripts to Python. Bash scripts are a pain to maintain; the `xqt` function in `ci_utils` makes it easy to run programs in a Python script with bash-like ease.

Reply:
I find bash scripts to be quite easy to write, read,  and maintain.

Most of them are for my use in maintaining and running Academy